tests/test_smart_ptr.py

```diff
@@ -29,11 +32,15 @@ def test_smart_ptr(capture):
             m.print_myobject1_2(o)
             m.print_myobject1_3(o)
             m.print_myobject1_4(o)
-        assert capture == "MyObject1[{i}]\n".format(i=i) * (4 if isinstance(o, int) else 8)
+        assert capture == "MyObject1[{i}]\n".format(i=i) * (
+            4 if isinstance(o, int) else 8
+        )
 
     cstats = ConstructorStats.get(m.MyObject1)
     assert cstats.alive() == 0
-    expected_values = ['MyObject1[{}]'.format(i) for i in range(1, 7)] + ['MyObject1[7]'] * 4
```

Comment:
What's the line length limit? This doesn't seem thát long?

Reply:
I left it completely default, which is 80 chars +10% (so max around 88). This is one of the _very_ few things you are allowed to configure, though. Configuration is against Black's concept, but that is one that can be changed.

Reply:
I used to set this to 120, because I liked that, but then got unhappy comments from people with lots of panes open that didn't have space for 120 chars. :/